Source/WTF/wtf/WorkQueue.h

```diff
@@ -86,23 +86,25 @@ class WorkQueueBase : public FunctionDispatcher, public ThreadSafeRefCounted<Wor
  * Runnables dispatched to a WorkQueue are required to execute serially.
  * That is, two different runnables dispatched to the WorkQueue should never be allowed to execute simultaneously.
  * They may be executed on different threads but can safely be used by objects that aren't already threadsafe.
- * Use `assertIsCurrent(m_myQueue);` in a runnable to assert that the runnable runs in a specific queue.
+ * Use `m_myQueue->assertIsCurrent();` in a runnable to assert that the runnable runs in a specific queue.
```

Comment:
No, this is not the design.
The design is:
 - if something can be asserted as being current, it responds to assertIsCurrent(something). That something can be anything: thread, threadlikeassertion, runloop, workqueue, etc. So all have the same uniform interface.

Reply:
assertIsCurrent(m_myQueue) will continue to work just the same now that WorkQueue is a SerialFunctionDispatcher ; the assertIsCurrent implementation is in FunctionDispatcher.h

Reply:
I'll update the comment then.
But all supports `something->assertIsCurrent()` I believe